mount.c

```diff
@@ -119,6 +119,27 @@ static void update_mount_flags_and_options(unsigned long *mountflags, char *opts
 	*newopts = 0;
 }
 
+void do_mount(const char *source, const char *target, const char *type, unsigned long flags, const char *options)
+{
+	if (mount(source, target, type, flags, options) == -1) {
+		err(1, "mount_entries: mount(\"%s\", \"%s\", \"%s\", %lu, \"%s\")",
+				source ? source : "null",
+				target,
+				type ? type : "null",
+				flags,
+				options ? options : "null");
+	}
+
+	/* Special case: we can't just do read-only bind mounts in a single step.
+	   instead, we have to remount it with MS_RDONLY afterwards. */
+
+	int ro_bind = (flags & (MS_BIND | MS_RDONLY)) == (MS_BIND | MS_RDONLY)
```

Comment:
`unsigned long`

Reply:
int is right in this case, this is a boolean condition

Reply:
Right, the `==`, of course :facepalm: